src/envConfig.ts

```diff
@@ -265,5 +265,6 @@ export const asEnvConfig = asObject({
       port: asOptional(asString, '8008')
     }),
     { host: 'localhost', port: '8008' }
-  )
+  ),
+  REMOTE_CONFIG_FETCH_INTERVAL: asOptional(asNumber, 12 * 60 * 60 * 1000) // 12 hours
```

Comment:
This seems VERY different from the prior default. Why 12 hours vs 1s?

Reply:
The prior wasn't the default. I had it at 1s during development, I will remove the 1s altogether from the commit